ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/CommandlineParser.java

```diff
@@ -61,9 +63,11 @@ public boolean parseCommandLine(final String... args) {
       handleParameterException(ex);
     } catch (final ExecutionException ex) {
       commandLine.usage(output);
+    } catch (final TransactionSignerInitializationException ex) {
+      // perform no-op.
```

Comment:
No logging of the error cause or rethrow?

Reply:
This has been handled in the exception handler below - so no, not at this point.